### PR TITLE
#8157: Update leaky_relu sweep config and doc

### PR DESCRIPTION
**Pull request by mcw-anasuya**
### Ticket
#8157 

### Problem description
Documentation incomplete for leaky_relu op with supported params

### What's changed
Updated doc and moved corresponding sweep test files

### Checklist
- [x] [Post commit CI passes](https://github.com/tenstorrent/tt-metal/actions/runs/11183327531) - PASSED
- [x] [Run Sweeps](https://github.com/tenstorrent/tt-metal/actions/runs/11235033810) - PASSED

### CODE REVIEW

**Review by VirdhatchaniKN — APPROVED** [review]

**Review by Aswinmcw — APPROVED** [review]
LGTM

**Review by mouliraj-mcw — APPROVED** [review]

**Review by eyonland — APPROVED** [review]

**Review by umadevimcw — APPROVED** [review]

**Review by jdesousa-TT — APPROVED** [review]

**Review by eyonland — APPROVED** [review]

File: tests/sweep_framework/sweeps/eltwise/unary/leaky_relu/leaky_relu.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    input_a_dtype,
+    input_a_layout,
+    input_a_memory_config,
+    output_memory_config,
+    negative_slope,
+    *,
+    device,
+) -> list:
+    data_seed = random.randint(0, 20000000)
+    torch.manual_seed(data_seed)
```
[the remark this answers is not in the document]
Comment (mcw-anasuya) [reply tests/sweep_framework/sweeps/eltwise/unary/leaky_relu/leaky_relu.py:61]: Yes, @TT-BrianLiu 
I've removed `random.seed(0)`

[the remark this answers is not in the document]
Comment (TT-BrianLiu) [reply tests/sweep_framework/sweeps/eltwise/unary/leaky_relu/leaky_relu.py:61]: Remove `data_seed` as well.

[the remark this answers is not in the document]
Comment (mcw-anasuya) [reply tests/sweep_framework/sweeps/eltwise/unary/leaky_relu/leaky_relu.py]: @TT-BrianLiu 
Removed.

File: tests/sweep_framework/sweeps/eltwise/unary/leaky_relu/leaky_relu.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+from tests.ttnn.utils_for_testing import check_with_pcc, start_measuring_time, stop_measuring_time
+from tests.sweep_framework.utils import gen_shapes
+from tests.tt_eager.python_api_testing.sweep_tests.generation_funcs import gen_func_with_cast_tt
+from models.utility_functions import torch_random
+
+# Override the default timeout in seconds for hang detection.
+TIMEOUT = 30
+
+
```
Comment (TT-BrianLiu) [inline tests/sweep_framework/sweeps/eltwise/unary/leaky_relu/leaky_relu.py:20]: Remove this
